src/cpu/aarch64/jit_uni_softmax.cpp

```diff
@@ -665,6 +665,283 @@ struct jit_softmax_t<sve_512> : public jit_softmax_base_t<sve_512> {
     }
 }; // namespace aarch64
 
+template <>
+struct jit_softmax_t<sve_256> : public jit_softmax_base_t<sve_256> {
```

Comment:
Is there a reason for repeating the code for SVE256 - which is the same as SVE512 from Lines 389? Can there be just one jit_softmax_t<isa> and then there be entries for SVE512, 256 etc? (See https://github.com/oneapi-src/oneDNN/pull/1445)

Reply:
Ok fair enough

Reply:
Thanks @cfRod